erpnext/selling/report/payment_terms_status_for_sales_order/payment_terms_status_for_sales_order.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) 2013, Frappe Technologies Pvt. Ltd. and contributors
+# License: MIT. See LICENSE
+
+import frappe
+from frappe import _, qb, query_builder
+from frappe.query_builder import functions
+
+
+def get_columns():
+	columns = [
+		{
+			"label": _("Sales Order"),
+			"fieldname": "name",
+			"fieldtype": "Link",
+			"options": "Sales Order",
+			"read_only": 1,
+		},
+		{
+			"label": _("Submitted"),
+			"fieldname": "submitted",
+			"fieldtype": "Date",
+			"read_only": 1
+		},
+		{
+			"label": _("Payment Term"),
```

Comment:
```suggestion
			"label": _("Payment Terms Template"),
```

Reply:
This field denotes the individual term in the template. I think, labelling it 'Payment Terms Template' would cause confusion.